addons/mail/static/src/core/public_web/store_service_patch.js

```diff
@@ -0,0 +1,25 @@
+import { patch } from "@web/core/utils/patch";
+import { Store } from "@mail/core/common/store_service";
+
+/** @type {import("models").Store} */
+const StorePatch = {
+    onStarted() {
+        super.onStarted(...arguments);
+        this.env.bus.addEventListener(
```

Comment:
Why move this to the store instead of `discuss_core_public_web_service`? It doesn't exist in 17.4, but it does in master. I would expect Discuss notifications to be handled by the Discuss service, as is done in other bundles.

Reply:
I prefer we stop this non-sense of having so many services, the name are jokingly long, let's keep amount of item small by patching store.